force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -204,9 +245,10 @@ public with sharing class FlowControlLoopsCollections {
         List<Integer> numberList = new List<Integer>{1, 2, 3, 4, 5}; //DO NOT CHANGE
 
         // Remove the 3rd element (number 3) from the list
+        numberList.remove(Integer.valueOf(2)); //Removes the element with value 3
```

Comment:
You can just use 2 directly. "2" is always an integer. It would only be a Decimal if you add a decimal point.